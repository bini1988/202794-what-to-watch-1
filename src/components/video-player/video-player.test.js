import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

it(`VideoPlayer correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <VideoPlayer/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
