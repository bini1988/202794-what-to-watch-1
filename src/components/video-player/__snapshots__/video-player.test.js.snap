// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders default markup 1`] = `
<video
  preload="none"
/>
`;
